packages/shared/src/components/profile/ProfileButton.tsx

```diff
@@ -14,40 +14,38 @@ const ProfileMenu = dynamic(
 
 interface ProfileButtonProps {
   className?: string;
-  atMobileSidebar?: boolean;
+  settingsIconOnly?: boolean;
 }
 
 export default function ProfileButton({
   className,
-  atMobileSidebar,
+  settingsIconOnly,
 }: ProfileButtonProps): ReactElement {
   const { isOpen, onUpdate, wrapHandler } = useInteractivePopup();
   const { user } = useContext(AuthContext);
 
   return (
     <>
-      {atMobileSidebar ? (
+      {settingsIconOnly ? (
         <Button
-          iconOnly
-          className="btn btn-tertiary"
+          variant={ButtonVariant.Tertiary}
           onClick={wrapHandler(() => onUpdate(!isOpen))}
           icon={<SettingsIcon />}
         />
       ) : (
         <SimpleTooltip placement="left" content="Profile settings">
-          <button
-            type="button"
+          <Button
+            variant={ButtonVariant.Float}
+            size={ButtonSize.Small}
             className={classNames(
-              'items-center p-0 ml-0.5 font-bold no-underline rounded-lg border-none cursor-pointer text-theme-label-primary bg-theme-bg-secondary typo-callout focus-outline',
-              className ?? 'flex',
+              'items-center !p-0 ml-0.5 text-theme-label-primary gap-2',
```

Comment:
This might be an exception because it's so custom. what's the default color we add on float buttons?

Reply:
The default is the same as the bell icon on the notifications button. So it would be greyish - #a8b3cf.

Reply:
Here is another response from design, we should keep it white: https://dailydotdev.slack.com/archives/C0381BQUEN9/p1702476773654429?thread_ts=1702475717.398979&cid=C0381BQUEN9

As for keeping it as a button or not, I think keeping the Button has better accessibility than having a `<span with onClick>`, e.g. for screen readers it's much easier if it's a `<button` if it's a clickable element. So my vote is for keeping it as a button.

Reply:
Ah yes but I mean using a `button` rather then our `Button` component?
I do think this is where `none` variant should actually be used almost.

Reply:
Let's do that, it avoids issues down the line if we make buttonv2 even more strict etc.